theme/open/views/lib/default.hbs

```diff
@@ -41,13 +41,13 @@
 	{{/block}}
 
 	{{#block "styles"}}
-	<link rel="preload" href="/fonts/ptsans/ptsans-regular.woff2" as="font" type="font/woff2" crossorigin>
-	<link rel="preload" href="/fonts/ptsans/ptsans-bold.woff2" as="font" type="font/woff2" crossorigin>
-	<link rel="preload" href="/fonts/ptsans/ptsans-italic.woff2" as="font" type="font/woff2" crossorigin>
-	<link rel="preload" href="/vendor-optimized/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0" as="font"
+	<link rel="preload" href="{{getVersionedPath '/fonts/ptsans/ptsans-regular.woff2'}}" as="font" type="font/woff2" crossorigin>
+	<link rel="preload" href="{{getVersionedPath '/fonts/ptsans/ptsans-bold.woff2'}}" as="font" type="font/woff2" crossorigin>
+	<link rel="preload" href="{{getVersionedPath '/fonts/ptsans/ptsans-italic.woff2'}}" as="font" type="font/woff2" crossorigin>
+	<link rel="preload" href="{{getVersionedPath '/vendor-optimized/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0'}}" as="font"
```

Comment:
The used fonts are defined in `ptsans.css`. Since staticify does not update the content of these files, I would suggest to manually add a version query(`?v=1.0.0` ) to the font files and inside the css.

Reply:
I added a default cache for other assets

Reply:
The cache headers are not added for the fonts at the moment, but preloading the font using a different name, than the name defined in css results in loading the fonts two times. I fixed that.